### ISSUE TITLE
Implementation on Event based on Event Registration

### ISSUE THREAD

**Issue by sgoEma** [sgoEma/EventManagement#6]
- I should be able to track revenue for an event. 
- I should be able to track the number of people who have registered for an event 
- If an attandee registers for a Dreamforce event , they should also be registered automatically for the Welcome sesion under this event.

**Comment by t-gul** [comment]
There was no Welcome session created before this task. Will we create it or wait for another task(not created  before). @sgoEma @sercanEmakina

**Comment by sgoEma** [comment]
Yes, you need to create a "Welcome" session record for the test purpose. @t-gul
